### PR TITLE
Apply survey button design

### LINKED ISSUE #481: Apply session detail survey button design
## Overview (Required)
- This is not applied the design. We should use an orange button.
image
image

If you do this, We invite you to Zeplin 🎨 (Please note that Zeplin's font size is not designed.)
https://app.zeplin.io/project/5a327a64f0b20176887c556c/screen/5a66b9288218dc236d2581d0

### PR DESCRIPTION
**Pull request by ycrack**
## Issue
- close #481 

## Overview (Required)
- Add color, elevation and ripple effect.

### CODE REVIEW

**Comment by takahirom** [comment]
:eyes:

**Comment by takahirom** [comment]
Can you paste screenshot? 👀

**Comment by ycrack** [comment]
OK. show the ripple effect is running.
screenshot_1516961779

**Comment by takahirom** [comment]
Looks good 👍

**Comment by ycrack** [comment]
But I could not find the Survey Screen so nothing happens when you press it.

**Comment by takahirom** [comment]
Probably we should 2dp corner radius for Button. But I think it is enough 👍 
I create that issue
https://material.io/guidelines/components/buttons.html#buttons-style

**Comment by takahirom** [comment]
@ycrack 
Yes, That is collect. We are creating that future now 😉 
https://github.com/DroidKaigi/conference-app-2018/pull/474

File: app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -86,4 +86,17 @@
         <item name="android:elevation" tools:targetApi="21">2dp</item>
     </style>
 
+    <style name="GoToQuestionnaire">
```
Comment (takahirom) [inline app/src/main/res/values/styles.xml]: 👍

File: app/src/main/res/drawable/bg_go_to_questionaire.xml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (takahirom) [inline app/src/main/res/drawable/bg_go_to_questionaire.xml]: PreLollipop 👍

**Pull request merged**